package.json

```diff
@@ -43,6 +43,7 @@
     "chai": "^3.5.0",
     "enzyme": "^2.3.0",
     "jsdom": "^9.4.0",
+    "lodash": "4.14.1",
```

Comment:
Which version of npm/node are you using? In node 5, I was _only_ able to get the tests to pass with this devDependency.